Backend/app/dashboard/provider-dashboard/src/Domain/Action/ProviderPlatform/Operator/FleetManagement.hs

```diff
@@ -72,3 +75,24 @@ buildFleetOwnerRegisterReq merchantShortId opCity API.Types.ProviderPlatform.Ope
       email = Nothing,
       ..
     }
+
+postFleetManagementFleetUnlink :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> Kernel.Prelude.Text -> Environment.Flow Kernel.Types.APISuccess.APISuccess)
+postFleetManagementFleetUnlink merchantShortId opCity apiTokenInfo fleetOwnerId = do
+  checkedMerchantId <- merchantCityAccessCheck merchantShortId apiTokenInfo.merchant.shortId opCity apiTokenInfo.city
+  transaction <- SharedLogic.Transaction.buildTransaction (Domain.Types.Transaction.castEndpoint apiTokenInfo.userActionType) (Kernel.Prelude.Just DRIVER_OFFER_BPP_MANAGEMENT) (Kernel.Prelude.Just apiTokenInfo) Kernel.Prelude.Nothing Kernel.Prelude.Nothing SharedLogic.Transaction.emptyRequest
+  SharedLogic.Transaction.withTransactionStoring transaction $ do
+    Client.callOperatorAPI checkedMerchantId opCity (.fleetManagementDSL.postFleetManagementFleetUnlink) fleetOwnerId apiTokenInfo.personId.getId
+
+postFleetManagementFleetLinkSendOtp :: (Kernel.Types.Id.ShortId Domain.Types.Merchant.Merchant -> Kernel.Types.Beckn.Context.City -> ApiTokenInfo -> API.Types.ProviderPlatform.Operator.FleetManagement.FleetOwnerPhoneSearchReq -> Environment.Flow API.Types.ProviderPlatform.Operator.FleetManagement.FleetOwnerSearchRes)
+postFleetManagementFleetLinkSendOtp merchantShortId opCity apiTokenInfo req = do
+  checkedMerchantId <- merchantCityAccessCheck merchantShortId apiTokenInfo.merchant.shortId opCity apiTokenInfo.city
+  transaction <- SharedLogic.Transaction.buildTransaction (Domain.Types.Transaction.castEndpoint apiTokenInfo.userActionType) (Kernel.Prelude.Just DRIVER_OFFER_BPP_MANAGEMENT) (Kernel.Prelude.Just apiTokenInfo) Kernel.Prelude.Nothing Kernel.Prelude.Nothing (Kernel.Prelude.Just req)
+  SharedLogic.Transaction.withTransactionStoring transaction $ do
+    Client.callOperatorAPI checkedMerchantId opCity (.fleetManagementDSL.postFleetManagementFleetLinkSendOtp) apiTokenInfo.personId.getId req
```

Comment:
let's not store req because it contains sensitive data as mobile phone. We can store response instead, see `postFleetManagementFleetCreate` for example
spec:
```
  FleetOwnerPhoneSearchReq:
    - mobileNumber: Text
    - mobileCountryCode: Text

  FleetOwnerSearchRes:
    - fleetOwnerId: Id Person
    - name: Text
    - derive: "'HideSecrets"
```
code:

```
  transaction <- SharedLogic.Transaction.buildTransaction (Domain.Types.Transaction.castEndpoint apiTokenInfo.userActionType) (Kernel.Prelude.Just DRIVER_OFFER_BPP_MANAGEMENT) (Kernel.Prelude.Just apiTokenInfo) Kernel.Prelude.Nothing Kernel.Prelude.Nothing SharedLogic.Transaction.emptyRequest
  SharedLogic.Transaction.withResponseTransactionStoring transaction $ do .........
```

Reply:
Could you change to function to `Transaction.withResponseTransactionStoring` as I mentioned above?